config/rbac/role.yaml

```diff
@@ -61,6 +61,27 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterrolebindings
+  - clusterroles
+  - rolebindings
+  - roles
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - '*'
```

Comment:
Let's be more explicit with the verbs we need here, if possible.

Reply:
I think we do need `*` otherwise I believe I was getting exceptions, though maybe a list of all of the verbs is better? FWIW other operators that are in the certified repo are using `*` for this. (not that that is saying much though), but thought I'd note that.